.github/workflows/linters.yml

```diff
@@ -22,12 +22,20 @@ jobs:
 
   docker-lint:
     runs-on: ubuntu-latest
+    strategy:
```

Comment:
For my own curiosity: What does this do?

Reply:
you can define stop on fail or not...